File: Beginner/1019 - Time Conversion/1019.c
#include <stdio.h>
#include <math.h>

int main()
{

    int sec, min, hour;

    scanf("%i", &sec);

    hour = sec / 3600;
    sec %= 3600;
    min = sec / 60;
    sec %= 60;

    printf("%i:%i:%i\n", hour, min, sec);

    return 0;
}

File: Beginner/1019 - Time Conversion/1019.cs
using System; 

class URI {
    static void Main(string[] args) { 

        int sec = int.Parse(Console.ReadLine());
        
        int hour = sec / 3600;
        sec %= 3600;
        int min = sec / 60;
        sec %= 60;

        Console.WriteLine($"{hour}:{min}:{sec}");
    }
}